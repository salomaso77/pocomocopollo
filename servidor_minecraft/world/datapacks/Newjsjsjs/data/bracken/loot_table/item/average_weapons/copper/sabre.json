{
	"pools": [{
		"rolls": 1,
		"entries": [{
			"type": "item",
			"name": "minecraft:stone_sword",
			"functions": [{
				"function": "minecraft:set_name",
				"target": "item_name",
				"name": "Copper Sabre"
			},{
				"function": "minecraft:set_lore",
				"entity": "this",
				"lore": [{
					"translate": "",
					"color": "blue",
					"italic": false
				},{
					"translate": "When in Main Hand:",
					"color": "gray",
					"italic": false
				},{
					"translate": "5.5 Attack Damage",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "1.6 Attack Speed",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "+2.5% Movement Speed",
					"color": "dark_green",
					"italic": false
				},
				{
					"translate": "Bonus Sweeping Attack",
					"color": "aqua",
					"italic": false
				}],
				"mode": "insert"
			},{
				"function": "minecraft:set_components",
				"components": {
					"minecraft:item_model": "bracken:average_weapons/copper/sabre",
					"minecraft:max_damage": 175
				}
			},{
				"function": "minecraft:set_attributes",
				"modifiers": [{
					"attribute": "minecraft:movement_speed",
					"id": "bracken:copper_sabre",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				},{
					"attribute": "minecraft:attack_speed",
					"id": "bracken:copper_sabre",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"attribute": "minecraft:attack_damage",
					"id": "bracken:copper_sabre",
					"amount": 4.5,
					"operation": "add_value",
					"slot": "mainhand"
				}]
			},{
				"function": "minecraft:toggle_tooltips",
				"toggles": {
					"attribute_modifiers": false
				}
			},{
				"function": "set_damage",
				"damage": {
					"min": 0.48,
					"max": 0.93
				}
			}]
		}]
	}]
}