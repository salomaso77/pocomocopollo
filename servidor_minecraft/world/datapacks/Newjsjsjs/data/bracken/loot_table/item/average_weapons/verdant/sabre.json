{
	"pools": [{
		"rolls": 1,
		"entries": [{
			"type": "item",
			"name": "minecraft:wooden_sword",
			"functions": [{
				"function": "minecraft:set_name",
				"target": "item_name",
				"name": "Verdant Sabre"
			},{
				"function": "minecraft:set_lore",
				"entity": "this",
				"lore": [{
					"translate": "",
					"color": "blue",
					"italic": false
				},{
					"translate": "When in Main Hand:",
					"color": "gray",
					"italic": false
				},{
					"translate": "5.5 Attack Damage",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "1.6 Attack Speed",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "+2.5% Movement Speed",
					"color": "dark_green",
					"italic": false
				},
				{
					"translate": "Bonus Sweeping Attack",
					"color": "aqua",
					"italic": false
				}],
				"mode": "insert"
			},{
				"function": "minecraft:set_components",
				"components": {
					"minecraft:item_model": "bracken:average_weapons/verdant/sabre",
					"minecraft:max_damage": 500,
					"minecraft:enchantment_glint_override": false
				}
			},{
				"function": "set_enchantments",
				"enchantments": {
					"bracken:poisoning_fix": 1,
					"bracken:infused_mending_fix": 1
				}
			},{
				"function": "minecraft:set_attributes",
				"modifiers": [{
					"attribute": "minecraft:movement_speed",
					"id": "bracken:verdant_sabre",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				},{
					"attribute": "minecraft:attack_speed",
					"id": "bracken:verdant_sabre",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"attribute": "minecraft:attack_damage",
					"id": "bracken:verdant_sabre",
					"amount": 4.5,
					"operation": "add_value",
					"slot": "mainhand"
				},
				{
					"attribute": "minecraft:sweeping_damage_ratio",
					"id": "bracken:verdant_sabre",
					"amount": 2,
					"operation": "add_value",
					"slot": "mainhand"
				}]
			},{
				"function": "minecraft:toggle_tooltips",
				"toggles": {
					"attribute_modifiers": false
				}
			},{
				"function": "set_damage",
				"damage": {
					"min": 0.48,
					"max": 0.93
				}
			}]
		}]
	}]
}