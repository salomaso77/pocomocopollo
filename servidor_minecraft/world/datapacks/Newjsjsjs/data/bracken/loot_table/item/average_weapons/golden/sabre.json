{
	"pools": [{
		"rolls": 1,
		"entries": [{
			"type": "item",
			"name": "minecraft:golden_sword",
			"functions": [{
				"function": "minecraft:set_name",
				"target": "item_name",
				"name": "Golden Sabre"
			},{
				"function": "minecraft:set_lore",
				"entity": "this",
				"lore": [{
					"translate": "",
					"color": "blue",
					"italic": false
				},{
					"translate": "When in Main Hand:",
					"color": "gray",
					"italic": false
				},{
					"translate": "4 Attack Damage",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "1.6 Attack Speed",
					"color": "dark_green",
					"italic": false
				},{
					"translate": "+2.5% Movement Speed",
					"color": "dark_green",
					"italic": false
				},
				{
					"translate": "Bonus Sweeping Attack",
					"color": "aqua",
					"italic": false
				}],
				"mode": "insert"
			},{
				"function": "minecraft:set_components",
				"components": {
					"minecraft:item_model": "bracken:average_weapons/golden/sabre"
				}
			},{
				"function": "minecraft:set_attributes",
				"modifiers": [{
					"attribute": "minecraft:movement_speed",
					"id": "bracken:golden_sabre",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				},{
					"attribute": "minecraft:attack_speed",
					"id": "bracken:golden_sabre",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"attribute": "minecraft:attack_damage",
					"id": "bracken:golden_sabre",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},
				{
					"attribute": "minecraft:sweeping_damage_ratio",
					"id": "bracken:golden_sabre",
					"amount": 2,
					"operation": "add_value",
					"slot": "mainhand"
				}]
			},{
				"function": "minecraft:toggle_tooltips",
				"toggles": {
					"attribute_modifiers": false
				}
			},{
				"function": "set_damage",
				"damage": {
					"min": 0.48,
					"max": 0.93
				}
			}]
		}]
	}]
}